oauth2client/client.py

```diff
@@ -1816,6 +1817,7 @@ def __init__(self, client_id,
                  device_uri=oauth2client.GOOGLE_DEVICE_URI,
                  token_info_uri=oauth2client.GOOGLE_TOKEN_INFO_URI,
                  authorization_header=None,
+                 pkce=False,
```

Comment:
How can you add a parameter to this constructor without also adding an entry to the constructor's doc string's "Args:" section?